pkg/operator/controllers/storageaccounts/storageaccount_controller.go

```diff
@@ -111,18 +112,10 @@ func (r *Reconciler) Reconcile(ctx context.Context, request ctrl.Request) (ctrl.
 
 // SetupWithManager creates the controller
 func (r *Reconciler) SetupWithManager(mgr ctrl.Manager) error {
-	aroClusterPredicate := predicate.NewPredicateFuncs(func(o client.Object) bool {
-		return o.GetName() == arov1alpha1.SingletonClusterName
-	})
-	masterMachinePredicate := predicate.NewPredicateFuncs(func(o client.Object) bool {
-		role, ok := o.GetLabels()["machine.openshift.io/cluster-api-machine-role"]
-		return ok && role == "master"
-	})
-
 	return ctrl.NewControllerManagedBy(mgr).
-		For(&arov1alpha1.Cluster{}, builder.WithPredicates(aroClusterPredicate)).
-		Watches(&source.Kind{Type: &machinev1beta1.Machine{}}, &handler.EnqueueRequestForObject{}, builder.WithPredicates(masterMachinePredicate)). // to reconcile on master machine replacement
-		Watches(&source.Kind{Type: &machinev1beta1.MachineSet{}}, &handler.EnqueueRequestForObject{}).                                              // to reconcile on worker machinesets
+		For(&arov1alpha1.Cluster{}, builder.WithPredicates(predicate.And(predicates.AROCluster, predicate.GenerationChangedPredicate{}))).
+		Watches(&source.Kind{Type: &machinev1beta1.Machine{}}, &handler.EnqueueRequestForObject{}, builder.WithPredicates(predicates.MachineRoleMaster)).    // to reconcile on master machine replacement
+		Watches(&source.Kind{Type: &machinev1beta1.MachineSet{}}, &handler.EnqueueRequestForObject{}, builder.WithPredicates(predicates.MachineRoleWorker)). // to reconcile on worker machines
```

Comment:
[Borderline Rant - Not Blocking Anything :upside_down_face:  ] 

These 2 lines are very long. I know Go doesn't have a max line length but we live in a physical world with fixed screens.  A quick `grep '.\{120\}' -r ./pkg/*/*.go | wc -l` tells me we have > 1000 lines longer than 120.

- Have we ever considered adding a max line length?
- Has this ever bothered anyone else?
- Do I need to stop worrying and learn to love some line wrapping plugin???